### ISSUE TITLE
SNS delivery policy always re-applies

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#170]
_This issue was originally opened by @tj as hashicorp/terraform#6754. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

Seems like it's just another JSON formatting issue, but adding a delivery_policy will always plan as changed immediately after an apply. In this case:

``` hcl
resource "aws_sns_topic" "report_results" {
  name = "report_results"
  delivery_policy = <<EOF
{
  "http": {
    "defaultHealthyRetryPolicy": {
      "minDelayTarget": 20,
      "maxDelayTarget": 20,
      "numRetries": 10,
      "numMaxDelayRetries": 0,
      "numNoDelayRetries": 0,
      "numMinDelayRetries": 0,
      "backoffFunction": "linear"
    },
    "disableSubscriptionOverrides": false
  }
}
EOF
}
```

**Comment by egarbi** [comment]
This can be closed as per #724 I think

**Comment by bflad** [comment]
I'm going to close this per the comment above as that seems correct (fixed in Terraform 0.9.5+ or Terraform 0.10+ and any version of the AWS provider since the split), but please open a new issue if this is still occurring in recent versions of the AWS provider. Thanks!
